### PR TITLE
Feature/update gallery reducer schema

### PR DESCRIPTION
**Pull request by achaselittlefield**
The motivation behind this card was to update the schema for the gallery reducer. I reviewed the schema provided in the attached trello card and was able to update it to reflect the new desired schema. The initial issue was that the schema was outdated and needed to be updated in order to provide new data to be used in the weekly submissions function.

https://trello.com/c/l4R6fQJW/473-update-the-gallery-reducer-to-receive-the-data-from-the-new-schema

https://www.loom.com/share/5e3f149754d94cd28c81da2147fadbbc

### CODE REVIEW

**Comment by venegasgabby76** [comment]
good work on updating the reducer to match the schema. this will make working with the gallery easier and clearer.

File: src/state/reducers/galleryReducer.js
Comment (Aaron-TheCreator) [inline src/state/reducers/galleryReducer.js:16]: missing "HasDrawn" property, see https://github.com/Lambda-School-Labs/scribble-stadium-be/blob/main/data/seeds/007_Submissions.js

File: src/components/pages/WordCloud/WordCloud.js
Comment (Aaron-TheCreator) [inline src/components/pages/WordCloud/WordCloud.js]: how is this file related to this Trello card? https://trello.com/c/l4R6fQJW/473-update-the-gallery-reducer-to-receive-the-data-from-the-new-schema 
Is this PR atomic?

**Review by Aaron-TheCreator — CHANGES_REQUESTED** [review]
galleryReducer looks pretty great, needs a small change. i dont believe the wordcloud logic should be included in this PR unless I'm misunderstanding something, looks really cool though. not sure if any meaningful change was made to the README other than whitespace added. could this be a problem of using "git add ." instead of "git add :< filepath >" ??

File: src/components/pages/WordCloud/WordCloud.js
Comment (achaselittlefield) [reply src/components/pages/WordCloud/WordCloud.js]: I restored the file from main so it has all the changes recently made on main.

File: src/state/reducers/galleryReducer.js
Comment (Kyleswillard) [reply src/state/reducers/galleryReducer.js:16]: This is indeed missing.

**Review by casey-dietz — DISMISSED** [review]
Nice work adding the hasDrawn property, and updating the gallery reducer

File: README.md
Comment (Kyleswillard) [inline README.md:269]: Is there a reason for the additional dead space?

**Review by Kyleswillard — DISMISSED** [review]
Generally looks good beyond the dead space on the first file.

**Review by chukalicious — CHANGES_REQUESTED** [review]
Team C is currently working on the feature that would use these changes. BE is still working on the the router and schema that the FE will ultimately be using. Merging this PR should be put on hold while the BE completes their task. Once they do, team C's FE should review the PR and determine if any further changes should be made to the reducer to meet the user story's requirements. If any, request them, and then approve a merge. In the interest of having this merged I'm going to withdraw this request so that we can continue work on our feature. Team C will perform the necessary changes.

**Comment by ashtilawat** [comment]
> Team C is currently working on the feature that would use these changes. BE is still working on the the router and schema that the FE will ultimately be using. Merging this PR should be put on hold while the BE completes their task. Once they do, team C's FE should review the PR and determine if any further changes should be made to the reducer to meet the user story's requirements. If any, request them, and then approve a merge.

Are we good to go here?

**Review by chukalicious — DISMISSED** [review]

**Review by casey-dietz — APPROVED** [review]
Nice work going in and completing some updates

**Review by Aaron-TheCreator — APPROVED** [review]
thank you Austin. this all looks good to me for the time being!

**Review by chukalicious — APPROVED** [review]
Thank you for you help with this!

**Pull request merged**
